gcp-jobs/furnishings/src/furnishings/services/reports/registrar_meta.py

```diff
@@ -60,7 +60,15 @@ class RegistrarInfo:
             "signatureImage": "registrar_signature_5.png",
             "signatureImageAndText": "registrar_signature_and_text_5.png",
             "startDate": "2025-04-18T00:00:00",
-            "endDate": None
+            'endDate': '2025-11-18T23:59:59'
+        },
+        {
+            'name': 'KERRY TAYLOR',
+            'title': 'Registrar of Companies',
+            'signatureImage': 'registrar_signature_6.png',
+            'signatureImageAndText': 'registrar_signature_and_text_6.png',
+            'startDate': '2025-11-19T00:00:00',
+            'endDate': None
```

Comment:
double quote instead of single quote

Reply:
These are Pacific times, not UTC, right?